app/View/Users/login.ctp

```diff
@@ -39,6 +39,9 @@
         <?php
             echo $this->Form->input('email', array('autocomplete' => 'off', 'autofocus'));
             echo $this->Form->input('password', array('autocomplete' => 'off'));
+            if (Configure::read('LinOTPAuth') == true) {
```

Comment:
Would change it to:
```php
if (!empty(Configure::read('LinOTPAuth'))) {
```
Same for the ones below

Reply:
Done